site/src/routes/package/[name]/components/primitive-name-tooltip.tsx

```diff
@@ -219,7 +219,7 @@ export function createPrimitiveNameTooltips(props: {
   primitives: BundlesizeItem[];
 }): void {
   onMount(() => {
-    const els = props.target.querySelectorAll<HTMLElement>("h2 code, a code");
+    const els = props.target.querySelectorAll<HTMLElement>("[data-code-primitive-name]");
```

Comment:
Code tag that has primitive name textcontent also exists by itself, so this query, `"h2 code, a code"`, wouldn't be able to find it. That's why code element that contains primitive name should have data attribute to specify that this code element is a primitive code element, since other code elements contain other content such as code blocks. So that's why I added this to modify primitive code element to include attribute https://github.com/solidjs-community/solid-primitives/pull/363/commits/8ffc84f848bc011cd8bddd5608a9c4b086a7867f#diff-35ac96e21c02642fbc0b8e07e7a59c676a741bd8c39a8b61fcf1f0423fdcd40cR117-R123

Reply:
Edit: updated the regex here https://github.com/solidjs-community/solid-primitives/pull/363/commits/ab8b58b09bf6370c2be6d2833eaeb42c03729d91

Reply:
imo better to just target all `code` blocks. if it matches it will create a tooltip.
but it's fine let's just roll with this

Reply:
I'd rather make the route page a tiny bit larger by adding those attributes, then add more runtime on page load, i'm definitely overestimating but getting textContent on those potential multiple large code blocks could be bad

Reply:
Actually I double down, yeah the execution time of textContent on a code element that contains a code block will contain nested spans and converting those to a string is bad